source/_header.html.erb

```diff
@@ -55,30 +55,30 @@
 				<li role="separator" class="divider"></li> %>
        <li><a href="/team">The Team</a></li>
        <li role="separator" class="divider"></li>
-       <li><a href="/logos">Logos</a></li>
+       <li><a href="/logos">Brand Guidelines</a></li>
```

Comment:
Not blocking, maybe pedantic- "Branding" instead of "Brand Guidelines" ?

Reply:
I like that. Especially since we have "community guidelines."